fhttp/httprunner.go

```diff
@@ -182,6 +184,11 @@ func RunHTTPTest(o *HTTPRunnerOptions) (*HTTPRunnerResults, error) {
 	// unused ones. We also must cleanup all the created clients.
 	keys := []int{}
 	for i := 0; i < numThreads; i++ {
+		// Get the report on the IP address each thread use to send traffic
+		ip := httpstate[i].client.GetIPAddress()
+		log.Infof("[%d] Host %s resolve to IP address: %s \n", i, o.URL, ip)
```

Comment:
- no trailing space before \n and no \n anyway in log.*

- o.URL isn't the host so either put just
`"[%d] %s resolved to: %s"`

or get the host part

Reply:
updated